rust/tests/integration/consumer.rs

```diff
@@ -974,7 +966,7 @@ fn dump_succeeds() {
                 vec![RtpEncodingParameters {
                     codec_payload_type: Some(100),
                     rtx: None,
-                    dtx: None,
+                    dtx: Some(false),
```

Comment:
Same question as in producer PR: why was this change necessary?

Reply:
It was not necessary and I'll put it back to `None`. So when the value coming from the server is `false` it will be `None` rather than `Some(false)`. I find the field type `Option<bool>` confusing in itself. Doesn't it make more sense for that such member `dtx` to just be `bool` and be defaulted to `false`?

Anyway I'm putting it back to `None`.

Reply:
`None` basically means it wasn't specified, use the default. We can make it just `bool`, but since it was optional I made it `Option`. While here `false` is the default, for some options the default for `None` is `true`.